quality/postman/sor-1-_table-table.postman_environment.json

```diff
@@ -1,44 +1,49 @@
 {
-	"id": "29d85e3c-308d-44f2-b24d-79e3c72f2751",
-	"name": "/sor/1/_table/:table",
-	"values": [
-		{
-			"key": "api_key",
-			"value": "local_admin",
-			"enabled": true
-		},
-		{
-			"key": "api_key_no_rights",
-			"value": "should_be_passed_as_param_in_command_line",
-			"enabled": true
-		},
-		{
-			"key": "baseurl_dc1",
-			"value": "localhost:8080",
-			"enabled": true
-		},
-		{
-			"key": "baseurl_dc2",
-			"value": "localhost:8080",
-			"enabled": true
-		},
-		{
-			"key": "write_consistency_weak",
-			"value": "WEAK",
-			"enabled": true
-		},
-		{
-			"key": "write_consistency_strong",
-			"value": "STRONG",
-			"enabled": true
-		},
-		{
-			"key": "write_consistency_nodurable",
-			"value": "NO_DURABLE",
-			"enabled": true
-		}
-	],
-	"_postman_variable_scope": "environment",
-	"_postman_exported_at": "2021-02-02T09:35:23.091Z",
-	"_postman_exported_using": "Postman/7.36.1"
-}
+  "_": {
+    "postman_variable_scope": "environment",
+    "postman_exported_at": "2021-02-02T09:35:23.091Z",
+    "postman_exported_using": "Postman/7.36.1"
+  },
+  "id": "29d85e3c-308d-44f2-b24d-79e3c72f2751",
+  "name": "/sor/1/_table/:table",
+  "values": [
+    {
+      "type": "any",
+      "value": "local_admin",
+      "key": "api_key"
+    },
+    {
+      "type": "any",
+      "value": "smx7thxearnabbc9iisj55dglbosj739bpfdorpxmzktyjyt",
```

Comment:
shouldn't this be passed from cmd?

Reply:
no, because they are prerequisites for the test environment that Postman uses, then this whole file is passed to Newman / cmd

Reply:
right, prerequisites step creates this key for different envs
this should be reverted

Reply:
this key has been deleted